pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: dart_style
 # Note: See tool/grind.dart for how to bump the version.
-version: 2.1.0
+version: 2.1.1
```

I think we want `2.2.0` for the corrected constraint right?
```suggestion
version: 2.2.0
```